qemu/tests/balloon_memhp.py

```diff
@@ -30,6 +31,7 @@ def check_memory():
         Check guest memory
         """
         if params['os_type'] == 'windows':
+            time.sleep(2)
```

Comment:
If we always need wait a few seconds after hotplug memory, shouldn't we change it in the vt, https://github.com/avocado-framework/avocado-vt/blob/master/virttest/utils_test/qemu/__init__.py#L709

Reply:
Agree with @huangyum, it will become more flexible.

Reply:
Somehow, I didn't hit any issue when running memory hotplug test cases with windows guest. 
Is it some issue with both balloon and memory hotplug?

Reply:
Which windows guest do you run? I hit this issue on win2016 and win2012.

Reply:
All kinds of windows data center version are covered in memory hotplug test cases. I didn't run them recently, but I never hit the issue before.

Reply:
As you said your test cases are about hotplug/unplug memory without balloon operation before hotplug.
For this issue, first run balloon qmp command to evict memory, then hotplug 1G new memory device,after 1s checking memory in guest.
I think it's normal sometimes that the guest can't get the correct value in 1s, in my tests, add 1-3s sleep time before check memory, it always can get correct memory value.